SCClassLibrary/Common/Audio/BasicOpsUGen.sc

```diff
@@ -2,16 +2,13 @@
 
 BasicOpUGen : UGen {
 	var <operator;
+	implicitResourceConnectionStrategies { ^[] }
+	hasObservableEffect { ^false }
+	canBeReplacedByIdenticalCall { ^true }
+	nameForDisplay { ^(super.name ++ rate ++ operator).asSymbol }
```

Comment:
I don't see `nameForDisplay` elsewhere. Should it exist for all UGens?

Reply:
(correction: I saw now that every UGen knows this message.)
The `nameForDisplay` is used as an efficient key `displayName` for a UGen in `visited`, combining name and rate. Why "display"? Isn't it more like an identifier?